src/pages/home.html

```diff
@@ -0,0 +1,1926 @@
+                            <path
+                                d="M18 12.375C18 13.5685 17.5259 14.7131 16.682 15.557C15.8381 16.4009 14.6935 16.875 13.5 16.875C12.3065 16.875 11.1619 16.4009 10.318 15.557C9.47411 14.7131 9 13.5685 9 12.375"
+                                stroke="#00ADB5"
+                                stroke-width="2"
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                            />
+                        </svg>
+
+                        <span
+                            class="bg-light-blue absolute -bottom-3 -right-3 flex h-[27px] w-[27px] items-center justify-center rounded-full text-lg text-[#fff]"
+                            >۲</span
+                        >
+                    </a>
+                </section>
+            </section>
+            <hr
+                class="mx-auto my-[30px] h-[0.6px] max-w-7xl rounded border-0 bg-mystic"
+            />
+
+            <nav
+                class="mx-auto flex w-full max-w-7xl items-center justify-between gap-10"
+            >
+                <section class="flex items-center justify-center gap-[50px]">
+                    <a
+                        href="#?"
+                        class="relative flex items-center justify-center gap-[6px]"
+                    >
+                        <span
+                            class="absolute -bottom-[30px] -left-4 h-[100px] w-[0.6px] bg-mystic"
+                        ></span>
+                        <img
+                            src="/src/assets/images/header/category.png"
+                            alt="ایکون دسته بندی"
+                            class="h-[25px] w-[24px]"
+                        />
+                        <p class="text-lg text-black">دسته بندی محصولات</p>
+                    </a>
+                    <a href="#?" class="flex items-center justify-center gap-2">
+                        <img
+                            src="/src/assets/images/header/gift.png"
+                            alt="ایکون هدیه"
+                            class="h-[24px] w-[24px]"
+                        />
+                        <p class="text-lg text-black-pearl">هدیه بدهید</p>
+                    </a>
+                    <a href="#?" class="flex items-center justify-center gap-2">
+                        <img
+                            src="/src/assets/images/header/tag.png"
+                            alt="ایکون تگ"
+                            class="h-[24px] w-[24px]"
+                        />
+                        <p class="text-lg text-black-pearl">پلن های پیشنهادی</p>
+                    </a>
+                    <a href="#?" class="text-lg text-lynch">شاسی ها</a>
+                    <a href="#?" class="text-lg text-lynch">تابلو ها</a>
+                    <a href="#?" class="text-lg text-lynch">تماس با ما</a>
+                    <a href="#?" class="text-lg text-lynch">درباره ما</a>
```

Comment:
Pay more attention to details. You can always use semi-space to look Persian words much better.

```suggestion
                    <a href="#?" class="text-lg text-lynch">شاسی‌ها</a>
                    <a href="#?" class="text-lg text-lynch">تابلوها</a>
                    <a href="#?" class="text-lg text-lynch">تماس با ما</a>
                    <a href="#?" class="text-lg text-lynch">درباره ما</a>
```